src/workerd/api/actor-state.h

```diff
@@ -348,11 +348,34 @@ class DurableObjectState: public jsg::Object {
   jsg::Promise<jsg::Value> blockConcurrencyWhile(jsg::Lock& js,
       jsg::Function<jsg::Promise<jsg::Value>()> callback);
 
+  void acceptWebSocket(jsg::Ref<WebSocket> ws, kj::Array<kj::String> tags);
```

Comment:
Is the intent here that at least one `tag` is always required? For instance, is `acceptWebSocket(ws, [])` acceptable? If it is ok to *not* provide tags, then I would make this `jsg::Optional<kj::Array<kj::String>> tags` so that devs can omit that second argument.

Reply:
Good question! I don't see any mention of this in the design doc, but since we also have `getWebSockets(tag)` I figure we would want at least one (maybe implicit) tag?

Ex. `getWebSockets("")` gives back all websockets accepted with one of:
- `acceptWebSocket(ws)`
- `acceptWebSocket(ws, [])`
- `acceptWebSocket(ws, [""])`

I wouldn't want to provide `getWebSockets()` (no tag) because as a user I would expect that to return all tags rather than the empty string tag, but even `getWebSockets("")` would feel a bit strange if I accepted the websocket without a tag in the first place...

Will ask in the API doc; happy to share the link over chat if you don't have access and want to add anything!

Reply:
Handled here
- https://github.com/cloudflare/workerd/pull/449/commits/ae4987ba9f2ef7b236e198e6e6e607deff9b88d1
- https://github.com/cloudflare/workerd/pull/449/commits/cdf825249136cc455c0896c19b489a971e2f4d99